### DISCUSSION TITLE
Problems running podman on Digital Ocean Droplet under Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by mwoodpatrick** [containers/podman#14151] [General]
I'm new to podman (but am experienced with docker). I'm trying to run podman on a Digital Ocean Droplet under Ubuntu 22.04, I have installed podman and  podman version is showing:

```
Version:      3.4.4
API Version:  3.4.4
Go Version:   go1.17.3
Built:        Wed Dec 31 16:00:00 1969
OS/Arch:      linux/amd64
```

I was having problems building an image

	podman build -t nginx https://git.io/Jf8ol
	STEP 1/10: FROM debian:buster-slim
	Resolved "debian" as an alias (/etc/containers/registries.conf.d/shortnames.conf)
	Trying to pull docker.io/library/debian:buster-slim...
	Getting image source signatures
	Copying blob 4be315f6562f done
	Error: error creating build container: writing blob: adding layer with blob "sha256:4be315f6562fccf08fd6c749557e31e45ab6d987370e20e2c4933ddb04ddd5ff": Error processing tar file(exit status 1): potentially insufficient UIDs or GIDs available in user namespace (requested 0:42 for /etc/gshadow): Check /etc/subuid and /etc/subgid: lchown /etc/gshadow: invalid argument

Is this expected? 

I got further around by adding 

`podman --storage-opt ignore_chown_errors=true build -t nginx https://git.io/Jf8ol`

But now I get

`E: gnupg, gnupg2 and gnupg1 do not seem to be installed, but one of them is required for this operation`

However, when I try to install it I see

```
sudo apt-get install gnupg
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
gnupg is already the newest version (2.2.27-3ubuntu2).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
```

It appears to be installed as gpgv

Doing 

`sudo ln -s /usr/bin/gpgv /usr/bin/gnupg2`

Does not help the issue, Any suggestions on how to fix

**Comment by rhatdan** [comment]
Do you have entries in /etc/subuid and /etc/subgid files?  If you modified them you need to do a 
`podman system migrate`

**Comment by mwoodpatrick** [comment]
Yes I do 

```
cat /etc/subuid
mwoodpatrick:100000:65536

 cat  /etc/subgid
mwoodpatrick:100000:65536
```

I had run 

` podman system migrate`

but it did not help

I would appreciate any pointers in best way to debug this

**Comment by mheon** [comment]
Please provide the full output of `podman info`

**Comment by mwoodpatrick** [comment]
I just figured out my problem (wrong username in files), many thanks for your help
